src/layout/Footer.tsx

```diff
@@ -15,39 +18,283 @@ export const Footer = () => {
         padding: '1rem',
         bgcolor: 'info.light',
         display: 'grid',
-        gridTemplateColumns: { xs: '1fr', lg: 'auto auto auto auto' },
-        gap: '0.5rem 1rem',
-        alignItems: 'center',
+        gridTemplateColumns: { xs: '1fr', lg: '1fr 1fr 1fr' },
```

Comment:
Syns kanskje den slår om til mobilvisning litt vel tidlig? Syns det passer best med `sm` her, men da blir det en liten bug med horisontal scrollbar som må løses også
```suggestion
        gridTemplateColumns: { xs: '1fr', sm: '1fr 1fr 1fr' },
```